config/foreman.migrations/20160607133050_foreman_proxy_puppetssh_rename.rb

```diff
@@ -0,0 +1,3 @@
+# Rename foreman_proxy provider "puppetssh" to "ssh"
+# http://projects.theforeman.org/issues/15323
+answers['foreman_proxy']['puppetrun_provider'] = 'ssh' if answers['foreman_proxy'] && answers['foreman_proxy']['puppetrun_provider'] == 'puppetssh'
```

Comment:
Yes, it'll be unset with the default answers file, but it'll still work as `Hash#[]` returns `nil`, and `nil` can be safely be compared.